### PR TITLE
Feature agent calendar

### PR DESCRIPTION
**Pull request by peterhsieh**
- AgentCalendar (View, Dialog, Delete)
- Events Resource API/Endpoints

### CODE REVIEW

**Comment by peterhsieh** [comment]
## Issues
- Couldn't work out how to sticky the toolbar (attempted to use react.sticky.. but it failed)
- Not sure how to have the list load 'today' at a scrolled position (is that even possible if there aren't enough list items?)

## ToDo
- [ ] Implement the Edit button with Dialog
- [ ] Events in the past are faded (Usability)

File: scripts/components/CalendarListItem.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+  },
+
+  deleteEvent() {
+    Api.deleteEvent({
+      eventId: this.props.event.id,
+      callback: (err, response) => {
+        if (err) {
+        console.log(err);
+        return reply(err);
```
Comment (nirrek) [inline scripts/components/CalendarListItem.jsx]: `reply()` is a Hapi.js serverside function.... Not sure why it is being used here.

**Comment by nirrek** [comment]
To scroll today into view use something like this:

``` javascript
componentDidMount() {
  React.findDOMNode(this.refs.today).scrollIntoView();
}
```

You will have to make sure you place a ref="today" onto the green header bit of today.

As for scrolling to it if there is no scroll position: no there is no simple way to implement this on the web afaik. Would require you being able to scroll past the actual height of the content. So it would be doable, I just don't think you can do it easily with existing APIs.

**Comment by nirrek** [comment]
## Todo Before Merging:
- [x] Implement `scrollIntoView` for the current date.
- [x] Fade out past events
- [x] Add ability to edit an event
- [x] Add a snackbar `Event deleted` / `Event updated` notification when performing actions.

**Comment by peterhsieh** [comment]
**A file in the Kronos causes the building of Bundle.js to fail** - I think it may be crashing with webpack. 
Workaround is to remove all content within the file (react-kronos>nmodules>color> nmodules>color-string>nmodules>color-name>index.json

**Pull request merged**